src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/QCallHandles.cs

```diff
@@ -75,7 +75,10 @@ internal QCallModule(ref System.Reflection.Emit.ModuleBuilder module)
         internal QCallAssembly(ref System.Reflection.RuntimeAssembly assembly)
         {
             _ptr = Unsafe.AsPointer(ref assembly);
-            _assembly = assembly.GetUnderlyingNativeHandle();
+            if (assembly != null)
+                _assembly = assembly.GetUnderlyingNativeHandle();
+            else
+                _assembly = IntPtr.Zero;
```

Comment:
Two things:

1. Code golf
   ```suggestion
               _assembly = assembly?.GetUnderlyingNativeHandle() ?? IntPtr.Zero;
   ```

2. Should we change the signature to `internal QCallAssembly(ref System.Reflection.RuntimeAssembly? assembly)` ? /cc @jkotas

Reply:
It would be nice to make QCallAssembly, QCallModule and QCallTypeHandle look the same. Looks good to me otherwise.